main.go

```diff
@@ -99,6 +99,12 @@ func main() {
 			Value:  "",
 			EnvVar: "MQTT_KEY",
 		},
+		cli.StringFlag{
+			Name:   "client-id,i",
+			Usage:  "Location of a TLS certificate .pem file for the Mosquitto message broker",
```

Comment:
You might want to change the description.

Reply:
silly me. will do. thanks for reviewing